src/Plugin.php

```diff
@@ -1070,6 +1107,14 @@ public function unactivateAll()
         if (isset($_SESSION['glpimenu'])) {
             unset($_SESSION['glpimenu']);
         }
+
+        Event::log(
+            '',
+            Plugin::class,
+            4,
+            "setup",
+            __('All plugins have been disabled by the GLPI update process.')
```

Comment:
Anyone could call this method. We do not want people to blame update process if a custom script calls this method.
```suggestion
            __('All plugins have been disabled.')
```